docker/Dockerfile

```diff
@@ -41,7 +43,8 @@ RUN rm -f /etc/apt/apt.conf.d/docker-clean
 # see: https://gist.github.com/tiran/2dec9e03c6f901814f6d1e8dad09528e
 # use space efficient utility from base image
 RUN --mount=type=cache,target=/var/cache/apt \
-    echo "deb https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy main" > /etc/apt/sources.list.d/deadsnakes-ppa.list && \
+    UBUNTU_CODENAME=$(. /etc/os-release && echo "$VERSION_CODENAME") && \
```

Comment:
We also want to update the keys to not use the globally trusted one, don't we (as per ehrql)?